styles.css

```diff
@@ -527,6 +514,131 @@ ul {
     color: #5e6c84;
     border: 1px solid #c1c7d0;
   }
+
+  #popup {
+    display: none;
+  }
```

Comment:
- [ ] I think you may remove this line code and work around it if at all it is needful, I could see that you've already selected this particular element in line 522. You may remove it according to HTML/CSS best practises (YAGNI). ℹ️ **Note: this is applicable to line 927 as well. Thanks.**